homeassistant/components/sutro/sensor.py

```diff
@@ -0,0 +1,159 @@
+"""Sensor platform for Sutro."""
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_PARTS_PER_MILLION,
+    PERCENTAGE,
+    TEMP_FAHRENHEIT,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    ATTRIBUTION,
+    DOMAIN,
+    ICON_ACIDITY,
+    ICON_ALKALINITY,
+    ICON_BATTERY,
+    ICON_CHLORINE,
+    ICON_TEMPERATURE,
+    NAME,
+    VERSION,
+)
+from .entity import SutroEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Sutro sensors."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        [
+            AciditySensor(coordinator, entry),
+            AlkalinitySensor(coordinator, entry),
+            FreeChlorineSensor(coordinator, entry),
+            TemperatureSensor(coordinator, entry),
+            BatterySensor(coordinator, entry),
```

Comment:
All these classes seems to be replaceable by using `EntityDescription`s.

Reply:
Thanks for the tip! Do you by any chance have a pointer to any documentation on EntityDescriptions or how they should be used? I see a lot of examples throughout the code but didn't see any helpful references in the docs:
https://developers.home-assistant.io/search?q=entitydescription

Reply:
I don't believe we have direct documentation for it. But, we do have tons of examples :)